tests/mockDataTests/writeAllReviews.test.js

```diff
@@ -0,0 +1,36 @@
+const writeAllReviews = require('../../mockData/writeAllReviews.js');
+
+const testRestaurants = [
+  {
+    id: 101,
+    title: 'Mario\'s Magnificent Pasta',
+    foodType: 'Italian',
+    rating: 5,
+    price: '$$$',
+  },
+];
+
+test('writeAllReviews returns an array', () => {
+  const reviews = writeAllReviews(testRestaurants);
+  expect(Array.isArray(reviews)).toBe(true);
+});
+
+test('writeAllReviews should write 3-4 reviews for each restaurant', () => {
+  testRestaurants[0].rating = 4;
+  const reviews = writeAllReviews(testRestaurants);
+
+  expect(reviews.length).toBeGreaterThanOrEqual(3);
+});
+
+test('The ratings average to each restaurant\'s overall rating', () => {
```

Comment:
I'd maybe rename this test to include the average you'd expect it to be just so it's more clear what the test is doing

Reply:
Resolved by additional commit.